op.c

```diff
@@ -16281,8 +16281,7 @@ S_aassign_scan(pTHX_ OP* o, bool rhs, int *scalars_p)
         else
             if (o == effective_top_op)
                 effective_top_op = o->op_sibparent;
```

Comment:
Would it not be even clearer to also pull the `if` up on the same line as the `else`?